packages/round-manager/src/context/round/UpdateRoundContext.tsx

```diff
@@ -136,41 +156,37 @@ const _updateRound = async ({
     // }
 
     if (editedGroups.RoundFeePercentage) {
-      console.log("updating round fee percentage")
-      const arg = round.roundMetadata?.quadraticFundingConfig
-        ?.matchingFundsAvailable;
-      transactionBuilder.add(UpdateAction.UPDATE_ROUND_FEE_PERCENTAGE, [arg]);
+      console.log("updating round fee percentage");
+      const arg =
+        round.roundMetadata?.quadraticFundingConfig?.matchingFundsAvailable;
+      transactionBuilder.add("updateRoundFeePercentage", [arg]);
     }
 
     if (editedGroups.StartAndEndTimes) {
-      console.log("updating start and end times")
-      transactionBuilder.add(UpdateAction.UPDATE_ROUND_START_AND_END_TIMES, [
-        Date.parse(round?.applicationsStartTime.toString()) / 1000,
-        Date.parse(round?.applicationsEndTime.toString()) / 1000,
-        Date.parse(round?.roundStartTime.toString()) / 1000,
-        Date.parse(round?.roundEndTime.toString()) / 1000,
+      console.log("updating start and end times");
+      transactionBuilder.add("updateStartAndEndTimes", [
+        BigInt(Date.parse(round?.applicationsStartTime.toString()) / 1000),
```

Comment:
If I understand this correctly, this is taking the epoch (ms) representation of the date and dividing by 1000 to seconds. I would be fine by rounding to the nearest second to avoid BigInt errors. Is that the best solution here?